File: app/src/main/java/com/gordienoye/cellulardatanotifier/ConnectivityBroadcastReceiver.java
package com.gordienoye.cellulardatanotifier;


import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;


public class ConnectivityBroadcastReceiver extends BroadcastReceiver {

  @Override
  public void onReceive (Context context, Intent intent) {
    ConnectivityNotifier.updateNotificationState(context);
  }
}

File: app/src/main/java/com/gordienoye/cellulardatanotifier/ConnectivityBroadcastReceiver.kt
package com.gordienoye.cellulardatanotifier

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import com.gordienoye.cellulardatanotifier.ConnectivityNotifier.updateNotificationState

class ConnectivityBroadcastReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        updateNotificationState(context)
    }
}